doc/src/manual/documentation.md

```diff
@@ -81,7 +81,23 @@ As in the example above, we recommend following some simple conventions when wri
    ...
    """
    ```
-5. Include any code examples in an `# Examples` section.
+5. Provide hints to similar functions
+   
+   Sometimes there are functions of similar functionality, which are named quite different. To increase 
+   discoverability please provide a short list of these similar functions in a `See also: ` section.
+   
+   ```
+   See also: bar!, baz, baaz
```

Comment:
Perhaps write this as 

```See also: [`bar!`](@ref), [`baz`](@ref),```

etc. This will cause the identifier to be quoted and a link to its documentation will be generated in the HTML.